packages/veritone-react-common/src/components/DataTable/table.test.js

```diff
@@ -251,6 +253,36 @@ describe('Table', function () {
     );
   });
 
+  it('exposes row index and column key when table cell is clicked', function () {
+    const spy = sinon.spy();
+
+    class Container extends React.Component {
+      getRowData = i => {
+        return this.props.data[i]; // eslint-disable-line
+      };
+
+      render() {
+        return (
+          <Table
+            rowGetter={this.getRowData}
+            rowCount={initialData.length}
+            onCellClick={spy}
+          >
+            <Column dataKey="id" header="id" />
+          </Table>
+        );
+      }
+    }
+
+    const initialData = [{ id: 0 }, { id: 1 }];
+
+    const wrapper = mount(<Container data={initialData} />);
+
+    wrapper.find('td').last().simulate('click');
+    wrapper.update();
```

Comment:
i expect you don't need to call update here, but try it to be sure

Reply:
fixed